Source/p_enemy.c

```diff
@@ -2696,6 +2709,478 @@ void A_LineEffect(mobj_t *mo)
     }
 }
 
+//
+// [XA] New mbf21 codepointers
```

Comment:
How would you like if we moved all these new code pointers into their own separate source file (and header), say `p_mbf21.c`? This would look a bit more tidied up (I have done the same in Crispy for the classic MBF code pointers, too). Also, how about the new pointers in `p_pspr.c`?

Reply:
I tried this, but it quickly got ugly with all the includes and extern function declarations for other codepointers. Also now the differences with dsda-doom implementation are minimal and I think this is handy for future updates/bugfixes.

Reply:
Alright, thanks!